net/message/message.go

```diff
@@ -196,9 +196,6 @@ func HandleNodeMsg(node Noder, buf []byte, len int) error {
 		return errors.New("Unexpected size of received message")
 	}
 
-	str := hex.EncodeToString(buf[:len])
-	log.Debug("Received data len: ", len, "\n", str)
```

Comment:
It is useful log to debug TX&RX issue although it locate at data path,  so the debug log should be turn off in any high performance  requirement scenarios